spring-pulsar-docs/build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
 description = 'Spring Pulsar Docs'
 
 ext {
-	micrometerDocsVersion="1.0.0-SNAPSHOT"
+	micrometerDocsVersion="1.0.0-M7"
```

Comment:
The MM docs generator has actually moved into a single module under `1.0.0`. I will update this is in a follow on PR though.